src/pages/pu-vendors/index.js

```diff
@@ -0,0 +1,131 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { RemittanceSettingsRepository } from 'src/repositories/RemittanceRepository'
+import { useResourceQuery } from 'src/hooks/resource'
+import { useWindow } from 'src/windows'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import VendorsWindow from './windows/VendorsWindow'
+import { PurchaseRepository } from 'src/repositories/PurchaseRepository'
+import RPBGridToolbar from 'src/components/Shared/RPBGridToolbar'
+
+const PuVendors = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { stack } = useWindow()
+  const { platformLabels } = useContext(ControlContext)
+
+  const {
+    query: { data },
+    labels: _labels,
+    filterBy,
+    paginationParameters,
+    invalidate,
+    refetch,
+    access
+  } = useResourceQuery({
+    queryFn: fetchGridData,
+    endpointId: PurchaseRepository.Vendor.qry,
+    datasetId: ResourceIds.PuVendors
+  })
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50 } = options
+    const defaultParams = `_startAt=${_startAt}&_pageSize=${_pageSize}&_params=&_sortField=`
```

Comment:
remove 
`  const defaultParams = `_startAt=${_startAt}&_pageSize=${_pageSize}&_params=&_sortField=`
    var parameters = defaultParams`
    
and use ``_startAt=${_startAt}&_pageSize=${_pageSize}&_params=&_sortField=`` directly in request params